homework-submission/todo_react/src/TodoLists.js

```diff
@@ -0,0 +1,34 @@
+import React from "react";
+import "./TodoLists.css";
+
+
+
+export default class TodoLists extends React.Component {
+
+  render() {
+
+    let { description, deadline, done, id, handleClick, removeItem } = this.props;
```

Comment:
The `id` has the gray color in my `vscode` means it passed to this component with no use, perhaps cleaning up the file a bit for readability, what do you say? 🙂